contrib/kube-prometheus/README.md

```diff
@@ -1,35 +1,40 @@
 # kube-prometheus
 
-This repository collects Kubernetes manifests, dashboards, and alerting rules
-combined with documentation and scripts to provide single-command deployments
-of end-to-end Kubernetes cluster monitoring.
+This repository collects Kubernetes manifests, [Grafana](http://grafana.com/) dashboards, and
+[Prometheus rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/)
+combined with documentation and scripts to provide single-command deployments of end-to-end
+Kubernetes cluster monitoring with [Prometheus](https://prometheus.io/) (Operator).
 
 ## Prerequisites
 
-First, you need a running Kubernetes cluster. If you don't have one, follow the
-instructions of [bootkube](https://github.com/kubernetes-incubator/bootkube) or
-[minikube](https://github.com/kubernetes/minikube). Some sample contents of this
+First, you need a running Kubernetes cluster. If you don't have one, we recommend you create one
+with [Tectonic Installer](https://coreos.com/tectonic/docs/latest/). Despite the name,
+Tectonic Installer gives you also the choice to create a barebones Kubernetes cluster, without
+CoreOS' Tectonic technology. Otherwise, you can simply make use of
+[bootkube](https://github.com/kubernetes-incubator/bootkube) or
+[minikube](https://github.com/kubernetes/minikube) for local testing. Some sample contents of this
```

Comment:
maybe we can comment that other methods of cluster creation might need a little adaptation, but we're happy to accepts additional docs/PRs to add more compatibility